src/kedro_templar/core/utils.py

```diff
@@ -4,15 +4,18 @@
 import os
 from pathlib import Path
 from typing import Text, Dict
-from ..import settings
 
 
 log = logging.getLogger(__name__)
 
 
 def load_config(config_path: Text) -> Dict:
-    """Loads and parases a yaml config into a dict object."""
-    return yaml.full_load(open(config_path))
+    """Loads and parses a yaml config into a dict object."""
+    if config_path.lower().startswith("s3"):
```

Comment:
Maybe we should use something like s3://  to make sure it is a path and not some weirdly named config like s3_config.yaml